src/azul/service/manifest_service.py

```diff
@@ -460,6 +460,9 @@ def tuple_or_none(v):
     return v if v is None else tuple(v)
 
 
+type Cursor = tuple[str, str]
```

Comment:
Needs documentation, e.g. what are the tuple elements. Reconcile with SortKey from elasticsearch_service.py. Double check the assumption that the first element is always a string.